pkg/cmd/root/help_topic.go

```diff
@@ -73,6 +73,19 @@ var HelpTopics = map[string]map[string]string{
 			- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
 			- %[1]spluck <field> <list>%[1]s: collects values of a field from all items in the input
 			- %[1]sjoin <sep> <list>%[1]s: joins values in the list using a separator
+			- %[1]struncate <length> <input>%[1]s: truncates input longer than length and adds "..."
+			- %[1]stable: starts a table. optional arguments match text/tabwriter.NewWriter after first argument
```

Comment:
I appreciate the sentiment, but I don't think it's a good idea to expose the underlying table-rendering mechanism here (`tabwriter`) because then we would have a hard time switching to another renderer in the future. Instead, I would suggest accepting no arguments.

Reply:
Most of those provide some level of customization to how the table is formatted. I could instead document them individually, if not even reduce them. The main ones of interest I think are `minwidth` and `padchar`.

Reply:
...though, I suppose we could simply wait for user feedback asking for such controls before adding them. I'll remove the support for table args.